docs/v1/P4Runtime-Spec.mdk

```diff
@@ -878,6 +900,48 @@ message PkgInfo {
   string url = 8;
 }
```

Comment:
do you want to add a `repeated string` field for extra annotations on the package? I would be fine with that.

Reply:
I like the idea.

Reply:
@antoninbas Actually field 4 is already there for annotations. I'd forgotten I put it there 10 months ago.